app/initialize.js

```diff
@@ -102,18 +102,20 @@ document.addEventListener('DOMContentLoaded', function init() {
   axios.get('//api.github.com/orgs/nio-blocks/repos?per_page=100')
   .then(function(response) {
     var data = response.data;
-    var link_header = parse_link_header(response.headers.link);
-    if (link_header.next) {
+    if (response.headers.link) {
+      // 12/7/16 - response.headers.link does not exist with fewer than 100
+      // blocks. Check again when we have more than 100 blocks.
+      // Only then is paging necessary -- KD
```

Comment:
I don't think the code is an appropriate place for this kind of thing. If you want to set a reminder for this, use GitHub issues or something else. This part is a good comment though `response.headers.link does not exist with fewer than 100 blocks`. Just by looking at the code, it's not obvious why this check is here so that's good.